apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.ts

```diff
@@ -1533,4 +1565,18 @@ export class NewBranchTransferAddComponent implements OnInit, OnChanges, OnDestr
             }
         });
     }
+
+    /**
+     * Checks if branch exists
+     *
+     * @private
+     * @param {string} branchUniqueName
+     * @param {*} branches
+     * @returns {boolean}
+     * @memberof NewBranchTransferAddComponent
+     */
+    private branchExists(branchUniqueName: string, branches: any): boolean {
```

Comment:
Return type should be number as it returns the length of the array
```suggestion
    private branchExists(branchUniqueName: string, branches: any): number {
```